docs/src/examples/10_ADD_ANOTHER_EXAMPLE.jl

```diff
@@ -0,0 +1,13 @@
+# # FBA with ???
+
+# Let's starting with loading the models and packages.
+
+!isfile("e_coli_core.xml") &&
+    download("http://bigg.ucsd.edu/static/models/e_coli_core.xml", "e_coli_core.xml")
+
+using COBREXA, Tulip
+
+model = load_model("e_coli_core.xml")
+
+import Random
+Random.seed!(1) # for repeatability of random numbers below
```

Comment:
this probably shouldn't be here? (or is just a TODO? in that case pls name reasonably and mark it TODO. Or maybe show how to simulate crowding with smoment?)

Reply:
yep, just a place holder, I didn't want to rename all the files... But the TODO part is in the name: `10_ADD_ANOTHER_EXAMPLE.jl`

Reply:
pls leave it as 10_crowding and write that we should add how to simulate crowding, we'll solve this with the rest of docs later

Reply:
will do